### ISSUE TITLE
Manifest including curly braces in property value doesn't build

### ISSUE THREAD

**Issue by anssiv** [mingyaulee/Blazor.BrowserExtension#50]
For some historical reasons we have `browser_specific_settings.gecko.id` set to a guid with curly braces and build throws an exception `The file manifest.json is not well formatted. Each line can only contain one of the characters '{', '}', '[', ']'`. Could we somehow ignore the validation if the braces are in a property value?

```
"browser_specific_settings": {
  "gecko": {
    "id": "{2dac9f13-26f2-401e-af9e-9c5a3ababbb5}"
  }
}
```

**Comment by mingyaulee** [comment]
Yes we can whitelist this pattern, I don't have the time to fix this yet but if you would like to help out, I'm happy to point you in the right direction.

**Comment by anssiv** [comment]
Unfortunately I ran into other problems with the library (our content scripts didn't work in Firefox) so I had to develop my own solution to run Blazor with our project. So at the moment I neither have time to fix this.

**Comment by johnW-ret** [comment]
I wouldn't mind taking a look at it given some direction.

Just curious, any particular reason for manual parsing over implementing a ``JsonConverter<Manifest>`` originally? https://learn.microsoft.com/en-us/dotnet/api/system.text.json.serialization.jsonconverter-1

**Comment by mingyaulee** [comment]
That would be great. Most likely the change is only required [here](https://github.com/mingyaulee/Blazor.BrowserExtension/blob/main/src/Blazor.BrowserExtension.Build/Tasks/ExtensionManifest/ManifestParser.cs).

The reason for not using System.Text.Json is because last I checked Visual Studio still uses .Net Framework to run the build tasks, so if we were to use it, we will need to include the binaries and also its dependencies.

**Comment by mingyaulee** [comment]
This is now fixed in `v0.10.7`. I have switched to using System.Text.Json after realising that it is used in another build task as well.
